CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+project(libMCNP2CAD)
+
+# cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 2.8)
+
+set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -DHAVE_IGEOM_CONE" )
+
+# adjust compiler setting for Linux using gcc version 5.0 and higher
+if(CMAKE_SYSTEM_NAME MATCHES Linux AND CMAKE_COMPILER_IS_GNUCXX)
+  if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "5.0")
+    if(NOT CMAKE_CXX_FLAGS MATCHES _GLIBCXX_USE_CXX11_ABI)
+      set(CMAKE_CXX_FLAGS "-D_GLIBCXX_USE_CXX11_ABI=0 ${CMAKE_CXX_FLAGS}")
+    endif()
+  endif()
+endif()
+
+# Get the packages needed to use the SDK
+find_package(Cubit REQUIRED CONFIG)
+include_directories(${CUBIT_INCLUDE_DIRS})
+
+# These directories need to be defined by the user.
+include_directories(${IGEOM_INCLUDE_DIR})
+include_directories(${MOAB_INCLUDE_DIR})
```

Comment:
I had been linking to the MOAB includes for iBase.h, since iGeom uses iBase_EntityHandles to keep track of everything.  I can actually just take iBase.h from MOAB; it's only about 500 lines. I was trying to avoid copying files as much as possible, but it's probably worth it to avoid having to have MOAB installed as well if one isn't using the plugin's iGeom.